beep/conversion_schemas/biologic_conversion.yaml

```diff
@@ -6,7 +6,7 @@ data_columns:
   cycle number:
```

Comment:
@willpowelson @moorepatrick Is the cycle number where the cycle index being stored?

Reply:
no, cycle index is stored under the header `cycle_index` which we add to the files in the pipeline.

Cycle number is calculated by Biologic based on charge/discharge alternance and is not what we want.

Reply:
see https://github.com/TRI-AMDD/beep-sync/pull/17#issuecomment-870810296
for the discussion where we debugged this in BEEP-sync